wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -1012,6 +1061,144 @@ class SwiftGenerator private constructor(
     )
   }
 
+  private fun defaultFieldInitializer(
+    protoType: ProtoType,
+    defaultValue: Any,
+  ): CodeBlock {
+    val typeName = protoType.typeName
+    return when {
+      defaultValue is List<*> -> defaultValue.toArrayFieldInitializer(protoType)
+      defaultValue is Map<*, *> -> defaultValue.toMapFieldInitializer(protoType)
+      typeName == BOOL -> CodeBlock.of("%L", defaultValue)
+      typeName == INT32 -> defaultValue.toInt32FieldInitializer()
+      typeName == INT64 -> defaultValue.toInt64FieldInitializer()
+      typeName == UINT32 -> defaultValue.toUInt32FieldInitializer()
+      typeName == UINT64 -> defaultValue.toUInt64FieldInitializer()
+      typeName == FLOAT -> defaultValue.toFloatFieldInitializer()
+      typeName == DOUBLE -> defaultValue.toDoubleFieldInitializer()
+      typeName == STRING -> CodeBlock.of("%S", stringLiteralWithQuotes2(defaultValue.toString()))
+      typeName == DATA -> CodeBlock.of(
+        "Foundation.Data(base64Encoded: %S)!",
+        defaultValue.toString().encode(charset = Charsets.ISO_8859_1).base64()
+      )
+      protoType.isEnum -> CodeBlock.of("%T.%L", typeName, defaultValue)
+      else -> throw IllegalStateException("$protoType is not an allowed scalar type")
+    }
+  }
+
+  private val Char.isIsoControl: Boolean
```

Comment:
Maybe add big `// region TEMPORARY` `// endregion` markers for the stuff that depends on that SwiftPoet issue (and linking to the GH issue)?

Reply:
good idea had thought of the same including a link to my PR fix.

Reply:
done.